libs/types/src/cdf/ballot-definition/vx-schema.json

```diff
@@ -63,6 +63,13 @@
         "IssuerAbbreviation": {
           "type": "string"
         },
+        "Office": {
+          "items": {
+            "$ref": "#/definitions/BallotDefinition.Office"
+          },
+          "minItems": 0,
+          "type": "array"
+        },
```

Comment:
We revived this from the original schema?

Reply:
Yep, just copy-pasted in from `nist-schema.json`, which lives in this repo